flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/ElasticsearchSinkFunction.java

```diff
@@ -69,4 +69,9 @@
 	 * @param indexer request indexer that {@code ActionRequest} should be added to
 	 */
 	void process(T element, RuntimeContext ctx, RequestIndexer indexer);
+
+	/**
+	 *  General-purpose open method.
```

Comment:
Initialization method for the function. It is called before the actual working `process` methods.